### ISSUE TITLE
Helmet and leggings disappear on invsee

### ISSUE THREAD

**Issue by Superalex0102** [Zrips/CMI#5821]
**Description of issue:**

If you use /invsee on another player twice, the leggings and the helmet just disappears.
https://gyazo.com/65ef5e15246fc4c7728aa668c3a3f9a4
---
**Cmi Version (using`/cmi version`):** 

```
[23:31:31 INFO]: --------------------------------------------------
[23:31:31 INFO]: CMI: 8.8.5.0 BungeeCord  SqLite-> 9.0.0.7
[23:31:31 INFO]: CMILib: 1.0.0.2 -> 1.0.2.9
[23:31:31 INFO]: Server: Paper(77cce823) 1.12.2-R0.1-SNAPSHOT
[23:31:31 INFO]: CMI economy: bekapcsolva Vault: 1.7.3-b CMI Chat: kikapcsolva
[23:31:31 INFO]: Modules -> 33 enabled 23 disabled: spawnerProximity, mirror, launchPad, tablist, elytraExploit, moneyCheque, portals, skin, holograms, elytraBoost, spawners, paintingEditor, playerChatTag, elytraLaunch, ranks, playerCombat, votifier, shulkerBackpack, headDrop, spawnerCharge, silentChest, namePlates, totemBehavior
[23:31:31 INFO]: --------------------------------------------------
```

**Server Type (Spigot/Paperspigot/etc):** 

PaperSpigot

**Server Version (using `/ver`):** 

```
[23:32:02 INFO]: This server is running Paper version git-Paper-77cce823 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
[23:32:02 INFO]: Previous version: git-Paper-1618 (MC: 1.12.2)
```

**Comment by mrfloris** [comment]
CMI 9 has basically replaced CMI 8, with 1.17.1 out and soon 9.0.0.7 getting an update to support this. I recommend upgrading to the latest version so any bugs fixed between CMI 8 and CMI 9 are live for you. Please test if this resolves this particular issue (which doesn't happen on my servers btw when i had 8850/1.16 or 9007/1.17

- backup your data prior to doing any upgrades, and test.

**Comment by Faceless0102** [comment]
> CMI 9 has basically replaced CMI 8, with 1.17.1 out and soon 9.0.0.7 getting an update to support this. I recommend upgrading to the latest version so any bugs fixed between CMI 8 and CMI 9 are live for you. Please test if this resolves this particular issue (which doesn't happen on my servers btw when i had 8850/1.16 or 9007/1.17
> 
> * backup your data prior to doing any upgrades, and test.

Yea, but it’s unstable for live servers, or not anymore?

**Comment by mrfloris** [comment]
No serious issues are being reported and the updates show small fixes, it seems to be pretty stable and spigot has posted that 1.17 is now their default revision. Kinda implying they think it's fine as recommended release. And now with 1.17.1 any serious 1.17 issues by mojang are fixed in 1.17.1 and paper 1.17.1, i think it's fine yes. i've moved all my live servers to 1.17.0 and just waiting for cmi to support 1.17.1
